ConfigurationSystem/Client/CSCLI.py

```diff
@@ -9,6 +9,10 @@
 import sys
 import signal
 import types
+import atexit
+import os
+import readline
```

Comment:
I confirm that readline is included in the Externals shipped with the DIRAC client and server.